README.md

```diff
@@ -20,6 +20,42 @@ Official Starting Page: https://dotnet.github.io
 * [Bringing more APIs to .NET Core](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/porting.md) (and why some APIs will be left out)
 
 
+### Daily Builds
+
+Daily builds of .NET Core components are published to [dotnet-core MyGet gallery](https://dotnet.myget.org/gallery/dotnet-core).
+The latest version number of each library can be seen in that gallery.
+
+Note: See officially supported [OS versions](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
+
+|    | Release |
```

Comment:
The content that these badges points to is not relevant to the target audience of coreclr/corefx repo README. We used to have badges like these in coreclr/corefx README.md before, and removed them because they were adding negative value (https://github.com/dotnet/coreclr/pull/7552 ). I do not think we should be adding them back to the README.

The README of the repo should focused on where to get the product that the repo participates in, how to report bugs and how to contribute changes.

Reply:
I agree. I don't mind few badges on README page as soon as they are at the bottom - the number of people who will leverage them is IMO small. How to use product, dogfood, file issues or start contributing PRs is IMO much higher value.

Reply:
It seems pretty common for github repos to put their badges on the main Readme. How about putting them at the end then?

Reply:
Can we have a single "Azure Pipelines" badge? I have looked at README pages of several popular projects and the common pattern that I see is that they have a single badge that does not clutter the page:

https://github.com/Microsoft/vscode
https://github.com/JamesNK/Newtonsoft.Json
https://github.com/emscripten-core/emscripten

Reply:
Fair enough. Can we put the full list on some linked secondary page for my own use if nothing else?

VS Code also somehow has badges for issues https://github.com/Microsoft/vscode/blob/master/README.md

Reply:
Having the full list on linked secondary page sounds good to me.

Reply:
So having one badge in the main page at the bottom and then link to a secondary page where we have all the badges?

Reply:
Sounds good. Where it goes on the page is up to you. We are probably over thinking 🙂

Reply:
I would prefer having just one badge for the builds at the top + other badges horizontally aligned to it, i.e. code coverage & codecov. Same as vscode: https://github.com/Microsoft/vscode/blob/master/README.md

Reply:
> I would prefer having just one badge for the builds at the top

I do like the "Azure Pipelines" badge that VSCode has. It is more of a statement that this project is using Azure DevOps than a link a lot of people are expected to click on. We should have that one.

> other badges horizontally aligned to it, i.e. code coverage & codecov. Same as vscode

Yes, I agree that vscode is a great example follow. The badges at the top of vscode README are chosen to be relevant to the target audience. vscode can have a lot more badges there, including a badge for code coverage reports, but they don't have them there for a reason.